parameter_sync.py

```diff
@@ -47,7 +50,7 @@ def process_parameter(param_name, param_value):
         new_file_full_path = temp_dir + os.sep + filename + '.new'
         logging.debug('Storing retrieved value for parameter "%s" in "%s"' % (param_name, new_file_full_path))
         with open(new_file_full_path, 'w') as f:
-            f.write(param_value.replace('\\n', '\n'))
+            f.write(param_value)
```

Comment:
Hmm this should still work with files that did have unescaped new lines so that's good.